.changeset/cyan-ants-pump.md

```diff
@@ -0,0 +1,6 @@
+---
+'example-app': patch
```

Comment:
```suggestion
```

Reply:
clarification question how does patch work in changeset?
I do yarn changeset select changed packages no on first major version select minor bump and summary ?
steps to do patch changeset?   
thank you